src/components/common/FileUploadPreview.vue

```diff
@@ -44,25 +39,28 @@ export default class FileUploadPreview extends Vue {
     fileUploadInput: HTMLFormElement
   }
 
-  private fileUpload: File = null
-  private customErrorMessages: string[] = []
+  // Local variables
+  fileUpload = null as File
+  customErrorMessages = [] as string[]
 
-  protected fileUploadRules = [
-    (v) => {
-      if (this.isRequired) {
-        return !!v || this.inputFileLabel + ' is required'
+  get fileUploadRules () {
+    return [
+      (file) => {
+        if (this.isRequired) {
+          return !!file || this.inputFileLabel + ' is required'
+        }
+        return true
+      },
+      (file) => {
+        if (this.maxSize) {
+          const maxSizeMB = this.maxSize / 1024
+          const errorMsg = 'Exceeds maximum ' + maxSizeMB.toString() + ' MB file size'
+          return (file?.size <= (this.maxSize * 1024)) || errorMsg
+        }
+        return true
       }
-      return true
-    },
-    (file) => {
-      if (this.maxSize) {
-        const maxSizeMB = this.maxSize / 1024
-        const errorMsg = 'Exceeds maximum ' + maxSizeMB.toString() + ' MB file size'
-        return (file?.size <= (this.maxSize * 1024)) || errorMsg
-      }
-      return true
-    }
-  ]
+    ]
+  }
```

Comment:
The old fileUploadRules array was static, so it didn't pick up changes due to different variable values.

The new fileUploadRules getter is reactive and now works properly.

Reply:
@ketaki-deodhar This is a code fix and it resolved the unit test errors for this component.